samples/nrf9160/nrf_cloud_rest_device_message/README.rst

```diff
@@ -0,0 +1,119 @@
+.. _nrf_cloud_rest_device_message:
+
+nRF9160: nRF Cloud REST Device Message
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The REST Device Message sample demonstrates how to use the `nRF Cloud REST API`_ to send `Device Messages <nRF Cloud Device Messages_>`_ using the ``SendDeviceMessage`` REST endpoint.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns
+
+The sample requires an `nRF Cloud`_ account.
+
+Your device must be provisioned with `nRF Cloud`_. If it is not, follow the instructions in `nrf_cloud_rest_device_message_provisioning`_.
+
+.. note::
+   This sample requires modem firmware v1.3.x or later.
+
+Limitations
+***********
+
+The `nRF Cloud REST API`_ requires all requests to be authenticated with a JSON Web Token (JWT).
+See  `nRF Cloud Security`_ for more details.
+Generating valid JWTs requires the network carrier to provide date and time to the modem, so the sample must first connect to an LTE carrier and determine the current date and time before REST requests can be sent.
+
+Note also that the `nRF Cloud REST API`_ is stateless.
+Devices may make REST requests at will.
```

Comment:
Really? Isn't it the user of the sample that is making the REST calls (requests)? Is this sentence needed at all when you compare REST to MQTT?

Reply:
Just to be clear, you are suggesting that I remove: `Devices may make REST requests at will.`

I think that could work; I wanted to elaborate a bit on what I meant by stateless, but I guess the comparison to MQTT does make that clear to a reader

Reply:
That was what I had in mind.